fluent-package/make-index-html.rb

```diff
@@ -0,0 +1,45 @@
+require 'erb'
+require 'find'
+require 'pathname'
+require 'optparse'
+
+options = {
+  :site => "https://fluentd.cdn.cncf.io",
+  :verbose => false
+}
+
+opt = OptionParser.new
+opt.on("-s", "--site [URL]") { |v| options[:site] = v }
```

Comment:
```suggestion
opt.on("-s", "--site URL") { |v| options[:site] = v }
```

`[]` makes the value optional. It means the value can be `nil`.
It appears that we should not allow `nil` here.